src/jdk.compiler/share/classes/com/sun/tools/javac/parser/JavacParser.java

```diff
@@ -62,11 +62,12 @@
 import static com.sun.tools.javac.resources.CompilerProperties.Fragments.VarAndExplicitNotAllowed;
 import static com.sun.tools.javac.resources.CompilerProperties.Fragments.VarAndImplicitNotAllowed;
 
-/** The parser maps a token sequence into an abstract syntax
- *  tree. It operates by recursive descent, with code derived
- *  systematically from an LL(1) grammar. For efficiency reasons, an
- *  operator precedence scheme is used for parsing binary operation
- *  expressions.
+/**
+ * The parser maps a token sequence into an abstract syntax tree.
+ * The current javac parser is a hand-written recursive-descent parser.
+ * It implements the grammar described in the Java Language Specification.
```

Comment:
In general, temporal words (like "current") should be avoided as a matter of style.

I suggest:

The parser is a hand-written recursive-descent parser that
implements the grammar described in the Java Language Specification.

Reply:
Fixed.